### PR TITLE
Register RDF namespaces, and update Blazegraph to 2.1.4

### PR DESCRIPTION
**Pull request by ruebot**
Addresses #333 and #382.

@Islandora-CLAW/committers to review. Please look closely at `claw.cnd` and make sure I've captured everything that is not already covered by `fedora-node-types.cnd`, and everything that was suggested in #333.

How to test: 
  - Verify Blazegraph comes up at http://localhost:8080/bigdata
  - Create a test resource, and hit it with `curl`, you should see something like the following:

```
[nruest@LB-SC-S1B4DB:install] (git)-[issue-333]-$ curl -i "http://localhost:8080/fcrepo/rest/TEST"
HTTP/1.1 200 OK
Server: Apache-Coyote/1.1
ETag: W/"d9cb04ed8d5201f0e72538d18facd26a560cf2cc"
Last-Modified: Tue, 27 Sep 2016 17:37:47 GMT
Link: <http://www.w3.org/ns/ldp#Resource>;rel="type"
Link: <http://www.w3.org/ns/ldp#Container>;rel="type"
Link: <http://www.w3.org/ns/ldp#BasicContainer>;rel="type"
Accept-Patch: application/sparql-update
Accept-Post: text/turtle,text/rdf+n3,text/n3,application/rdf+xml,application/n-triples,multipart/form-data,application/sparql-update
Allow: MOVE,COPY,DELETE,POST,HEAD,GET,PUT,PATCH,OPTIONS
Preference-Applied: return=representation
Vary: Prefer
Vary: Accept, Range, Accept-Encoding, Accept-Language
Content-Type: text/turtle
Content-Length: 2869
Date: Tue, 27 Sep 2016 17:37:55 GMT

@prefix schema: <http://schema.org> .
@prefix premis: <http://www.loc.gov/premis/rdf/v1#> .
@prefix bf: <http://id.loc.gov/ontologies/bibframe/> .
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix nt: <http://www.jcp.org/jcr/nt/1.0> .
@prefix pcdm: <http://pcdm.org/models#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix acl: <http://www.w3.org/ns/auth/acl#> .
@prefix pcdmfmt: <http://pcdm.org/2015/10/14/file-format-types> .
@prefix xsi: <http://www.w3.org/2001/XMLSchema-instance> .
@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
@prefix mode: <http://www.modeshape.org/1.0> .
@prefix pcdmuse: <http://pcdm.org/use#> .
@prefix xml: <http://www.w3.org/XML/1998/namespace> .
@prefix pcdmwrks: <http://pcdm.org/works#> .
@prefix jcr: <http://www.jcp.org/jcr/1.0> .
@prefix rel: <http://id.loc.gov/vocabulary/relators/> .
@prefix dcterms: <http://purl.org/dc/terms/> .
@prefix fedoraconfig: <http://fedora.info/definitions/v4/config#> .
@prefix mix: <http://www.jcp.org/jcr/mix/1.0> .
@prefix prov: <http://www.w3.org/TR/prov-o/> .
@prefix islandorarelsint: <http://islandora.ca/ontology/relsint#> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix cc: <http://creativecommons.org/ns#> .
@prefix image: <http://www.modeshape.org/images/1.0> .
@prefix ore: <http://www.openarchives.org/ore/terms/> .
@prefix sv: <http://www.jcp.org/jcr/sv/1.0> .
@prefix test: <info:fedora/test/> .
@prefix fedoramodel: <info:fedora/fedora-system =def/model#> .
@prefix gn: <http://www.geonames.org/ontology#> .
@prefix pcdmrts: <http://pcdm.org/rights#> .
@prefix islandorarelsext: <http://islandora.ca/ontology/relsext#> .
@prefix dwc: <http://rs.tdwg.org/dwc/terms/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix fedora: <http://fedora.info/definitions/v4/repository#> .
@prefix nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#> .
@prefix ebucore: <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
@prefix ldp: <http://www.w3.org/ns/ldp#> .
@prefix iana: <http://www.iana.org/assignments/relation/> .
@prefix xs: <http://www.w3.org/2001/XMLSchema> .
@prefix exif: <http://www.w3.org/2003/12/exif/ns#> .
@prefix dc: <http://purl.org/dc/elements/1.1/> .


<http://localhost:8080/fcrepo/rest/TEST> a fedora:Container , fedora:Resource ;
	fedora:lastModifiedBy "bypassAdmin"^^<http://www.w3.org/2001/XMLSchema#string> ;
	fedora:createdBy "bypassAdmin"^^<http://www.w3.org/2001/XMLSchema#string> ;
	fedora:created "2016-09-27T17:37:47.165Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> ;
	fedora:lastModified "2016-09-27T17:37:47.165Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> ;
	a ldp:RDFSource , ldp:Container ;
	fedora:writable "true"^^<http://www.w3.org/2001/XMLSchema#boolean> ;
	fedora:hasParent <http://localhost:8080/fcrepo/rest/> .
```

### CODE REVIEW

**Comment by whikloj** [comment]
One PR per ticket, sir.

Next time right 😉

File: install/configs/claw.cnd [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
+ * Islandora CLAW namespaces
+ */
+<acl = 'http://www.w3.org/ns/auth/acl#'>
+<bf = 'http://id.loc.gov/ontologies/bibframe/'>
+<cc = 'http://creativecommons.org/ns#'>
+<dcterms = 'http://purl.org/dc/terms/'>
+<dwc = 'http://rs.tdwg.org/dwc/terms/'>
+<exif = 'http://www.w3.org/2003/12/exif/ns#'>
+<fedoramodel = 'info:fedora/fedora-system =def/model#'>
```
Comment (peichman-umd) [inline install/configs/claw.cnd]: That ` =def` in the middle of the `fedoramodel` definition looks like a typo.

Comment (peichman-umd) [inline install/configs/claw.cnd]: :+1:

**Comment by whikloj** [comment]
`vagrant up`

**Comment by whikloj** [comment]
👍  looks good, test creation of a resource using JSON-LD data through Crayfish properly populated Blazegraph with the new version (2.1.4). 

I'll leave this until the call tomorrow incase there is any discussion around more prefixes/namespaces to register.

**Comment by dannylamb** [comment]
We didn't talk about this at the meeting, but I see no reason not to merge it.  Seems like a good improvement to me.  I say go for it @whikloj

**Pull request merged**